* Fri Jul 11 2014 Patrick Monnerat <pm@datasphere.ch>
- The "itk.tcl" possible list of pathes is not satisfactory for Linux (at
  lseat for Fedora). Extend the path list as for Darwin for all Unixes.

diff --git a/generic/itkBase.c b/generic/itkBase.c
index 79073ae..d344ce9 100644
--- a/generic/itkBase.c
+++ b/generic/itkBase.c
@@ -73,7 +73,7 @@ namespace eval ::itk {\n\
             lappend dirs [file join $bindir .. .. library]\n\
             lappend dirs [file join $bindir .. .. itk library]\n\
             # On MacOSX, check the directories in the tcl_pkgPath\n\
-            if {[string equal $::tcl_platform(platform) \"unix\"] && \
+            if {[string equal $::tcl_platform(platform) \"unix\"] || \
                     [string equal $::tcl_platform(os) \"Darwin\"]} {\n\
                 foreach d $::tcl_pkgPath {\n\
                     lappend dirs [file join $d itk$patchLevel]\n\
